spec/app/ics-028-cross-chain-validation/technical_specification.md

```diff
@@ -139,8 +140,13 @@ The CCV module is initialized through the `InitGenesis` method when the chain is
 - On the consumer chain, the genesis state is described by the following interface:
   ```typescript
   interface ConsumerGenesisState {
-    providerClientState: ClientState
-    providerConsensusState: ConsensusState
+    newChain: Bool // true for new consumer chain, false for chain restart
+    providerClientId: Identifier // empty for a new chain
+    providerChannelId: // empty for a new chain
+    providerClientState: ClientState // nil on restart
```

Comment:
Can you explain the two cases of an `NewChain` genesis and a restart genesis before this?

Reply:
Personally I don't think the restart genesis is necessary to put in the specification. It is more a quirk of the SDK than anything else

Reply:
I agree. I'll remove anything related to ExportGenesis and restarting a chain.